core/block_render_svg_vertical.js

```diff
@@ -974,6 +976,51 @@ Blockly.BlockSvg.prototype.renderDraw_ = function(iconWidth, inputRows) {
 };
 
 /**
+ * Give the block an attribute 'data-shapes' that lists its shape[s]
+ * @private
+ */
+Blockly.BlockSvg.prototype.renderClassify_ = function() {
+  var shapes = [];
+  
+  if(this.outputConnection) {
+    if(this.isShadow_) {
+      shapes.push('argument');
+    } else {
+      shapes.push('reporter');
+    }
+    if(this.edgeShape_ === Blockly.OUTPUT_SHAPE_HEXAGONAL) {
+      shapes.push('boolean');
+    } else if(this.edgeShape_ === Blockly.OUTPUT_SHAPE_ROUND) {
+      shapes.push('round');
+    }
+  } else {
+    // count the number of statement inputs
+    var inputList = this.inputList;
+    var statementCount = 0;
+    for (var i = 0, input; input = inputList[i]; i++) {
+      if (input.connection && input.connection.type === Blockly.NEXT_STATEMENT) {
+        statementCount++;
+      }
+    }
+    
+    if(statementCount) {
+      shapes.push('c-block');
```

Comment:
You mean the order of items in the list? I imagine this would be used with css's [a~=b] selector, which expects a space-separated list but shouldn't be affected by the order of the list. In JS, I imagine most use-cases would look for one list-item at a time so that shouldn't be an issue